.kitchen.vagrant.yml

```diff
@@ -0,0 +1,101 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: ansible_playbook
+  hosts: localhost
+  roles_path: ../
+  require_ansible_repo: false
+  require_ansible_omnibus: false
+  require_ansible_source: false
+  require_pip: true
+  ansible_version: 2.4.3.0
+  http_proxy: <%= ENV['HTTP_PROXY'] %>
+  https_proxy: <%= ENV['HTTPS_PROXY'] %>
+  no_proxy: localhost,127.0.0.1
+  ignore_extensions_from_root: [".git",".idea",".kitchen.yml"]
+  ignore_paths_from_root: [".git",".idea",".kitchen"]
+  <% if ENV['VERSION'] %>
+  attributes:
+    extra_vars:
+      es_major_version: "<%= ENV['VERSION'] %>"
+      <% if ENV['VERSION'] == '5.x' %>
+      es_version: '5.6.7'
+      <% end %>
+  <% end %>
+
+verifier:
+  name: inspec
+  sudo: true
+  inspec_tests:
+    - git: https://github.com/elastic/elasticsearch-inspec.git
+      branch: 6.0
+  attributes:
+    es_service_name: security_node_elasticsearch
+    elasticsearch_conf: /etc/elasticsearch/security_node/elasticsearch.yml
+    ssl_param: true
+    es_pass: elasticChanged
+    elastic_ip: localhost
+    es_logs_dir: /var/log/elasticsearch/localhost-security_node
+  format: json
+  output: ./%{platform}_%{suite}_results.json
+
+transport:
+  max_ssh_sessions: 6
+
+platforms:
+- name: centos-7
+  driver:
+    vagrantfiles:
+      - VagrantfileA.rb
+    name: vagrant
+    box: bento/centos-7
+    provision: true
+    pre_create_command: cp <%=ENV['ES_XPACK_LICENSE_FILE']%> {{vagrant_root}}
```

Comment:
Is there a reason this can't be mounted as a volume the same way it is being done for the docker based tests? Mostly because `cp` won't be available for anyone testing from a windows host. 

```
      volume: 
        - <%=ENV['ES_XPACK_LICENSE_FILE']%>:/tmp/license.json
```

Reply:
I'd have to look more into that I think. kitchen-docker and kitchen-vagrant differ in possible commands, I didn't find something similar to the kitchen-docker `volume:` one in kitchen-vagrant so I had to make do with the `pre_create_command:`.